src/commcare_cloud/ansible/roles/zookeeper/templates/myid.j2

```diff
@@ -0,0 +1,5 @@
+{% for host in groups['zookeeper'] %}
+{% if host == inventory_hostname %}
+{{ host }}
+{% endif %}
+{% endfor %}
```

Comment:
Is this the same thing as
```suggestion
{% if inventory_hostname in groups['zookeeper'] %}
{{ inventory_hostname }}
{% endif %}
```

and also by this point would this ever be running on a host that _isn't_ in the `[zookeeper]` inventory group? So is it the same thing as simply

```suggestion
{{ inventory_hostname }}
```

Reply:
yes. ` inventory_hostname` is fine :-)